src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react"
 import { BrowserRouter, Switch, Route } from "react-router-dom"
+import "bootstrap/dist/css/bootstrap.min.css"
```

Comment:
bootstrap นี่มาจากไหนนะครับ พอดีไม่เห็นว่าใน package install มายังไง
ใช้ bower ?

Reply:
ใน package.json ก็มีนะครับ ส่วนนี้เป็นของเพื่อนทำมา เข้าใจว่า yarn มาครับ
https://github.com/it-kmitl-2018/group3-frontend/blob/a344ac67c25f344f9036fbef512cb548a255a28c/package.json#L6

Reply:
เห็นละครับ พอดีดูผิดใน master